spec/ruby/library/fiber/transfer_spec.rb

```diff
@@ -11,7 +11,13 @@
   it "transfers control from one Fiber to another when called from a Fiber" do
     fiber1 = Fiber.new { :fiber1 }
     fiber2 = Fiber.new { fiber1.transfer; :fiber2 }
-    fiber2.resume.should == :fiber1
+
+    ruby_version_is '' ... '3.0' do
+      fiber2.resume.should == :fiber1
+    end
+    ruby_version_is '3.0' do
+      fiber2.resume.should == :fiber2
+    end
```

Comment:
It's better to have `ruby_version_is` outside of `it "description" do`, so the description can explain the difference in behavior. I can do that after this PR is merged.